open-trip-planner/src/main/scala/com/azavea/busplan/routing/StudentToStopRouting.scala

```diff
@@ -34,12 +36,13 @@ object StudentToStopRouting {
   }
 
   def getMaxDistance(maxDistance: Double, grade: Int): Double = {
+    var newMaxDistance = maxDistance;
     if (grade < 7) {
-      if (maxDistance > 2640) {
-        val maxDistance = 2640
+      if (maxDistance >= 804) {
+        newMaxDistance = 804
       }
     }
-    maxDistance
+    newMaxDistance
```

Comment:
`if` statements in Scala are expressions, so this could be rewritten as

```scala
def getMaxDistance(maxDistance: Double, grade: Int): Double =
	if(grade < 7 && maxDistance > 804) {
	  804
	} else {
 	  maxDistance
	}